### DISCUSSION TITLE
Any timelines for readiness of metrics?

### DISCUSSION THREAD

**Discussion by finalreq** [open-telemetry/opentelemetry-cpp#1468] [Q&A] [answered]
Hi all,

I was wondering, are there any estimates or targets for when metrics will be ready to use?
I'm looking to leverage OpenTelemetry for our project at Microsoft. I understand a timeline might not even exist but I thought I'd ask just so we have an idea of how far away metrics are from being production ready.

Thanks,
Armin

**Comment by lalitb** [comment, marked as answer]
The current Release Candidate plan is here - https://github.com/open-telemetry/opentelemetry-cpp/milestone/13. The v1.0 /GA release would be ~ one month from there.

**Reply by finalreq** [reply]
Perfect. Thank you!
